### DISCUSSION TITLE
Help with tp.file.include functionality

### DISCUSSION THREAD

**Discussion by celiaccb** [SilentVoid13/Templater#869] [Help]
Hi,
I am trying to use the following templater command work, in which filename is the name of a note located in the same folder as the note in which I am writing the templater command

```
<%+ await tp.file.include("[[filename]]") %>
```

I would expect that when I switch from live preview to reading mode, the line would be replaced by the filename content (since I used <%+), but when I switch to reading mode it keeps showing the templater command.

Where am I going wrong?

Thank you.

**Comment by AB1908** [comment]
Huh, for once this sounds like an actual bug. I will try to reproduce this.

**Reply by celiaccb** [reply]
Hi! Are there any updates on this?

**Reply by AB1908** [reply]
It moreso looks like this wasn't supported in the first place. I've also put up a request for deprecating dynamic commands in #913 as it's been very difficult to support and has caused several misunderstandings with many of our users. Do you want to chat about the use case so I can recommend other approaches?
